packages/core/src/helpers/color/color.ts

```diff
@@ -22,9 +22,8 @@ export function color(name: Color): string;
 export function color(name: Palette, opacity?: number): string;
 
 export function color(name: Color, opacity?: number): string {
-  return opacity
-    ? `rgba(var(--ods-color-${name}), ${opacity})`
-    : `rgba(var(--ods-color-${name}))`;
+  const alpha = opacity ? `, ${opacity}` : '';
+  return `rgba(var(--ods-color-${name})${alpha})`;
```

Comment:
I must say... it maybe cutting some size, but decrease readability.

Reply:
Yea... Would a helper make it more legible?

```ts
export function color(name: Color, opacity?: number): string {
  return `rgba(var(--ods-color-${name})${maybe(opacity)})`;
}

const maybe = (value: unknown) => (value ? `, ${value}` : '');
```

Or can you think of a different way?

It might be too early to optimise for size anyway.
If you're not happy with any approach, I don't mind reverting this.

Reply:
With helper looks little better. But anyhow I feel like original implementation matches Sass one better.

Reply:
We could refactor the one in Sass to be about the same, but I agree on the readability aspect.
Sass doesn't benefit from cutting implementation details size as it doesn't ship to the browser.

I'll just revert then, we can worry about that in the future, probably won't be our bottleneck anyway.
BTW we should probably include bundle size checks in our CI, like [bundlesize](https://github.com/siddharthkp/bundlesize) or [bundlewatch](https://github.com/bundlewatch/bundlewatch)

Reply:
Yeah, cool - please could you [add an issue](https://github.com/onfido/castor/issues).

Reply:
https://github.com/onfido/castor/issues/118